root/static/scripts/edit/URLCleanup.js

```diff
@@ -4782,9 +4782,17 @@ const CLEANUPS: CleanupEntries = {
               target: ERROR_TARGETS.ENTITY,
             };
           case LINK_TYPES.otherdatabases.recording:
+            if (prefix === 'release' && !(subPath === 'musicvideo')) {
```

Comment:
Is the current style because you expect to add more subpaths to the inner condition?
```suggestion
            if (prefix === 'release' && subPath !== 'musicvideo') {
```